README.md

```diff
@@ -1,4 +1,36 @@
-# eco-pass-kit
+<!-- 
+#################################################################################
+Tractus-X - Eco Pass KIT
+
+Copyright (c) 2022, 2023 Contributors to the Eclipse Foundation
+
+See the NOTICE file(s) distributed with this work for additional
+Information regarding copyright ownership.
+
+This program and the accompanying materials are made available under the
+terms of the Apache License, Version 2.0 which is available at
+
+https://www.apache.org/licenses/LICENSE-2.0.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+either express or implied. See the
+License for the specific language govern in permissions and limitations
+under the License.
+
+SPDX-License-Identifier: Apache-2.0
+################################################################################## 
+-->
+
+![Eco Pass Kit Pictotogram](/docs/resources/img/EcoPassKIT_pictogram.png)
+
+<h1 style="display:flex; align-items: center;">&nbsp;&nbsp;Eco Pass KIT</h1>
+
+[![Contributors][contributors-shield]][contributors-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Apache 2.0 License][license-shield]][license-url]
+[![Latest Release][release-shield]][release-url]
 
 This is the repository of ECO-Pass KIT. In this repository we collect all the information needed to use the this KIT.
```

Comment:
"repository of **the** EcoPass Kit"
information needed to use **"the" or "this"** KIT

Reply:
fixed